README.md

```diff
@@ -409,7 +409,7 @@ Emoji | Aliases
 <img src="img-buildkite-64/shipit.png" width="20" height="20" alt="shipit"/> | `:shipit:`
 <img src="img-buildkite-64/suspect.png" width="20" height="20" alt="suspect"/> | `:suspect:`
 <img src="img-buildkite-64/terminal.png" width="20" height="20" alt="terminal"/> | `:terminal:`, `:console:`
-<img src="img-buildkite-64/trollface.png" width="20" height="20" alt="trollface"/> | `:trollface:`, `:troll:`
+<img src="img-buildkite-64/trollface.png" width="20" height="20" alt="trollface"/> | `:trollface:`
```

Comment:
People are in for a surprise if they're using the existing one, I guess! I wonder if customers will care?

Reply:
But we can supersede unicode standard so removed our alias in https://github.com/buildkite/emojis/pull/369/commits/88e6bd40a607829b12c4a2c1deba54d69ade4349